widgets/clear-all/clear-all.js

```diff
@@ -0,0 +1,96 @@
+let React = require('react');
+let ReactDOM = require('react-dom');
+
+let utils = require('../../lib/utils.js');
+let bem = utils.bemHelper('ais-clear-all');
+let cx = require('classnames');
+
+let autoHideContainer = require('../../decorators/autoHideContainer');
+let headerFooter = require('../../decorators/headerFooter');
+
+let isEmpty = require('lodash/lang/isEmpty');
+
+let defaultTemplates = require('./defaultTemplates');
+
+/**
+ * Allows to clear all refinements at once
+ * @param  {string|DOMElement} options.container CSS Selector or DOMElement to insert the widget
+ * @param  {Object} [options.cssClasses] CSS classes to add
+ * @param  {string|string[]} [options.cssClasses.root] CSS class to add to the root element
```

Comment:
Yes, we need another issue for this, I think array css classes are not even working